backend/db/models/Question.js

```diff
@@ -0,0 +1,18 @@
+const connection = require('../connection')
+;
+const { Sequelize } = connection;
+const { UUID, UUIDV4, TEXT, STRING } = Sequelize;
+
+module.exports = connection.define('question', {
+  id: {
+    type: UUID,
+    primaryKey: true,
+    defaultValue: UUIDV4
+  },
+  text: {
+    type: TEXT
```

Comment:
Should probably add some checks/validations here (`allowNull`, `notEmpty`, etc.)